vector/src/main/res/layout/vector_room_header.xml

```diff
@@ -56,56 +56,66 @@
 
     <RelativeLayout
         android:id="@+id/action_bar_header_room_members_layout"
-        android:layout_below="@id/action_bar_header_room_topic"
-        android:layout_marginBottom="19dp"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content">
+        android:layout_height="wrap_content"
+        android:layout_below="@id/action_bar_header_room_topic"
+        android:layout_marginBottom="19dp">
 
         <!-- dummy text view to compute the text height -->
         <TextView
             android:id="@+id/action_bar_header_line_height"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:visibility="invisible"
+            android:text="measurement text"
```

Comment:
That hardcoded string should be removed. Not sure that view is still useful also.